library/testing/TestSDKCamera.cxx

```diff
@@ -313,4 +375,4 @@ int TestSDKCamera([[maybe_unused]] int argc, [[maybe_unused]] char* argv[])
   }
 
   return EXIT_SUCCESS;
-}
+}
```

Comment:
please do not remove end of line character (should be done in your IDE settings).